.vale/styles/RedHat/CaseSensitiveTerms.yml

```
@@ -76,6 +76,7 @@ swap:
   cidr|Classless Interdomain Routing|Classless Inter-domain Routing: CIDR
   classic mode: GNOME Classic
   CloudForms Management Engine|CFME: Red Hat CloudForms|Red Hat CloudForms Appliance
+  command prompt: Command Prompt|shell prompt
```

Without some context, folks might replace "command prompt" with "Command Prompt" everywhere. Would adding these parenthetical comments help?
```suggestion
  command prompt: Command Prompt (Windows)|shell prompt (Linux and elsewhere)
```

---

shell prompt first, then specify that Command Prompt is the name of a Windows product?

```suggestion
  command prompt: shell prompt|Command Prompt (Windows product)
```